### PR TITLE
Add EncryptionInfos & Logging for Maxtocode

### PR DESCRIPTION
**Pull request by Tianjiao**
- Partial Fix latest Maxtocode(3.87) (untested)
- Partial Fix some Maxtocode versions (untested)
- Improved very verbose logging for Maxtocode

### CODE REVIEW

File: 致中国区用户的一封信.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## **警告：不得删除本信！** ##
+
+*免责声明：本信仅代表个人观点，不作为专业法律建议，若有需要，请联系附近有资质的律师。*
+
+# 致中国区用户的一封信 #
+
+您好！欢迎您阅读本信。
+
+本软件是一款.NET反混淆与解密工具。我是本软件的开发者，Tianjiao。本软件是基于de4dot的修改版，原作者是网友0xd4d。
```
Comment (UlyssesWu) [inline 致中国区用户的一封信.md]: > 我是本软件的开发者
"I am the developer of this software" - NO, you're NOT.

> 原作者是网友0xd4d。
"The original author is *netizen* 0xd4d" - How dare you call 0xd4d as "网友"(netizen) ? He is a *real* developer, a *real* master on .NET. It's shame that you was not aware of that.

**Comment by UlyssesWu** [comment]
If I could have the right to handle this PR, I definitely **refuse** it just because I don't want the irrelevant "致中国区用户的一封信.md" (A Letter To Chinese Users) to be merged into this great repo, even if you have removed that in the latest commit. You should create a new branch and push clean commits. DO NOT let that upsetting "letter" appears anymore.

You may have awared - You have used incorrect descriptions - I've pointed some of them out in my review. And you have add some unclear & amateurish declarations about "Using this software may violate Chinese laws" etc. You can add such "letter" in your fork, but it's silly to pollute this original repo.

**Comment by Tianjiao** [comment]
Sorry for that.

I didn't intend to have this letter appeared in the original repo, I have reverted it.

I created the PullRequest branch in the Github Desktop, it brings ALL the commit history from my fork of master branch. It is a technical error.

Anyway, I will consider to create a new clean branch to pull this PR, because it is important that this obfuscator has not been updated nearly 4 years. You can blame me, but respect the users who needs de4dot to be fixed, even partially.

I still think that you misunderstand me. I am not careful in some wording, I confess. But I don't have any intention to disrespect. Chinese language is very subtle, different people have different opinions. If you want to know more about my letter, follow the links in it.

Sorry again.

**Comment by UlyssesWu** [comment]
Oh, come on, I do not think I "misunderstood" you. It's impossible for an honest coder to say he is "the developer of **this software**" despite the fact that he just added 150 lines of code in a huge project. (You should call yourself as a *contributer*. )

Every native Chinese speaker (including me) may consider you as the primary developer after reading your "letter". Your words are just disrespectful to Mr.0xd4d. That upsets me a lot, and I wish you could remove those lines in your repo as well.

**Pull request closed without merging**